utils/codepointtrie/benches/testdata/gc_for_test.toml

```diff
@@ -0,0 +1,853 @@
+  0xb,0xb,0xb,0xb,0xb,0,5,5,5,5,5,5,5,5,5,5,
+  5,5,5,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0x17,0x17,0x17,0x17,0,0,
+  0,0,0,5,8,8,8,8,8,8,8,8,8,8,8,8,
+  8,8,8,8,0,0,0,0,0,0,0,6,6,6,4,4,
+  4,4,4,4,4,4,4,4,4,4,4,0x17,4,6,0,0,
+  0,0,0,0,0,0,0,0,0,8,8,0,0,0,0,0,
+  0,0,0,0,0,0,0,0,0,4,4,4,4,0,4,4,
+  4,4,4,4,4,0,4,4,0,0,0,0,5,5,5,5,
+  0,0,0,0,0,0,0,0,5,5,5,5,5,5,5,5,
+  5,5,0,0,0x1b,6,6,0x17,0x10,0x10,0x10,0x10,0,0,0,0,
+  0,0,0,0,0,0,0,0,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0,
+  0,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,8,8,6,6,6,0x1b,0x1b,0x1b,
+  8,8,8,0x10,0x10,0x10,0x10,0x10,0x10,0x10,0x10,6,6,6,6,6,
+  0x1b,0x1b,6,6,6,6,6,6,6,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,
+  0x1b,0x1b,0x1b,6,6,6,6,0x1b,0x1b,6,6,6,0x1b,0,0,0,
+  0,0,0,0,0,0,0,0xb,0xb,0xb,0xb,0,0,0,0,0,
+  0,0,0,0,0,0,0,2,2,2,2,1,1,1,1,1,
+  1,1,1,1,1,1,1,2,2,2,2,2,2,2,2,2,
+  2,2,2,1,0,1,1,0,0,1,0,0,1,1,0,0,
+  1,1,1,1,0,1,1,1,1,1,1,2,2,2,2,0,
+  2,0,2,2,2,2,1,1,0,1,1,1,1,0,0,1,
+  1,1,1,1,0,1,1,1,1,1,1,1,0,2,2,2,
+  2,2,2,2,2,1,1,0,1,1,1,1,0,1,1,1,
+  1,1,0,1,0,0,0,1,1,1,1,1,1,0,2,2,
+  2,2,2,2,2,2,2,2,2,2,2,2,1,1,1,1,
+  2,2,2,2,2,2,0,0,1,1,1,1,1,1,1,1,
+  0x18,2,2,2,2,2,2,2,2,2,2,2,2,2,2,1,
+  1,1,1,1,1,1,1,1,1,1,0x18,2,2,2,2,2,
+  0x18,2,2,2,2,2,2,1,1,1,1,1,1,1,1,1,
+  0x18,2,2,2,2,2,2,0x18,2,2,2,2,2,2,1,2,
+  0,0,9,9,9,9,9,9,9,9,9,9,9,9,9,9,
+  9,9,6,6,6,6,6,6,6,0x1b,0x1b,0x1b,0x1b,6,6,6,
+  6,6,6,6,6,6,6,6,6,6,0x1b,0x1b,0x1b,0x1b,0x1b,6,
+  0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,6,0x1b,0x1b,0x17,0x17,0x17,
+  0x17,0x17,0,0,0,0,2,2,2,2,2,2,2,2,2,2,
+  2,2,2,2,2,0,6,6,6,6,6,6,6,6,6,0,
+  0,6,6,6,6,6,0,6,6,0,6,6,6,6,6,0,
+  0,0,0,0,6,6,6,6,6,6,6,4,4,4,4,4,
+  4,4,0,0,9,9,9,9,9,9,9,9,9,9,0,0,
+  0,0,5,0x1b,5,5,5,5,5,5,5,5,5,5,5,5,
+  5,5,6,0,9,9,9,9,9,9,9,9,9,9,0,0,
+  0,0,0,0x19,5,5,5,5,5,5,5,0,5,5,5,5,
+  0,5,5,0,5,5,5,5,5,0,0,0xb,0xb,0xb,0xb,0xb,
+  0xb,0xb,0xb,0xb,2,2,2,2,6,6,6,6,6,6,6,4,
+  0,0,0,0,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,
+  0x1b,0xb,0xb,0xb,0x19,0xb,0xb,0xb,0xb,0,0,0,0,0,0,0,
+  0,0,0,0,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,
+  0xb,0xb,0x1b,0xb,0,5,5,0,5,0,0,5,0,5,5,5,
+  5,5,5,5,0,5,5,5,5,0,5,0,5,0,0,0,
+  0,5,0,0,0,0,5,0,5,0,5,0,5,5,5,0,
+  5,5,0,5,0,0,5,0,5,0,5,0,5,0,5,5,
+  0,5,0,0,5,5,5,5,0,5,5,5,5,0,5,5,
+  5,5,0,5,0,5,5,5,0,5,5,5,5,5,0,5,
+  5,5,5,5,0x18,0x18,0,0,0,0,0,0,0,0,0,0,
+  0,0,0,0,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,0xb,
+  0xb,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1a,0x1a,0x1a,0x1a,
+  0x1a,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0,0,0,0,0,0x1b,0x1b,
+  0x1b,0x1b,0x1b,0,0,0,0x1b,0x1b,0x1b,0x1b,0x1b,0,0,0,0x1b,0x1b,
+  0x1b,0,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0x1b,0,0x10,
+  0,0,0,0,0,0,0,0,0,0,0,0,0,0,0x10,0x10,
+  0x10,0x10,0x10,0x10,0x10,0x10,0x10,0x10,0x10,0x10,0x10,0x10,0x10,0x10,0x11,0x11,
+  0x11,0x11,0x11,0x11,0x11,0x11,0x11,0x11,0x11,0x11,0x11,0x11,0,0,0,0
+]
+indexLength = 3404
+dataLength = 9984
+highStart = 0x110000
+shifted12HighStart = 0x110
+type = 1
```

Comment:
`question`: does it mean that this is using `TrieType` `small` rather than `fast`? Shouldn't we benchmark the fast one?

Reply:
Good catch; I'll add both small and fast